src/cli/dev.ts

```diff
@@ -60,7 +60,7 @@ export default function (program: RootCmd) {
           program = fork(path, {
             env: {
               ...process.env,
-              LOG_LEVEL: "debug",
+              LOG_LEVEL: "info",
```

Comment:
Seeing this in code, I'm kinda feeling like `pepr dev` _should_ still be debug by default... :thinking: 

Given it's use during development, what do we think of keeping this one as verbose by default?

Reply:
I agree, during dev debug makes sense

Reply:
Update made.